src/Compiler/CompilerOptions.cpp

```diff
@@ -198,6 +198,11 @@ llvm::cl::opt<bool> enableParallel("parallel",
                    "Set to 'true' if you want to enable parallelization."),
     llvm::cl::init(false), llvm::cl::cat(OnnxMlirOptions));
 
+llvm::cl::opt<bool> disableSimdOption("disable-simd",
+    llvm::cl::desc("Disable SIMD optimizations (default: on at O3, "
+                   "off otherwise). Set to `true` to disable SIMD at O3."),
```

Comment:
`default: on at O3` looks a bit confusing, in the sense that `on` is for this flag `disable-simd` or for `SIMD on`? Usually, `default` refers to default value for the current flag.

Reply:
I can improve the text.